metricbeat/module/audit/kernel/audit_linux.go

```diff
@@ -0,0 +1,303 @@
+package kernel
+
+import (
+	"os"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/pkg/errors"
+
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/logp"
+	"github.com/elastic/beats/libbeat/monitoring"
+	"github.com/elastic/beats/metricbeat/mb"
+	"github.com/elastic/beats/metricbeat/mb/parse"
+	"github.com/elastic/go-libaudit"
+	"github.com/elastic/go-libaudit/aucoalesce"
+	"github.com/elastic/go-libaudit/auparse"
+)
+
+const (
+	metricsetName = "audit.kernel"
+	logPrefix     = "[" + metricsetName + "]"
+
+	reassemblerMaxInFlight = 5
+	reassemblerTimeout     = 2 * time.Second
+	streamBufferLen        = 64
+)
+
+var (
+	debugf = logp.MakeDebug(metricsetName)
+
+	auditMetrics = monitoring.Default.NewRegistry(metricsetName)
+	lostMetric   = monitoring.NewInt(auditMetrics, "lost")
+
+	logBetaOnce sync.Once
+)
+
+func init() {
+	if err := mb.Registry.AddMetricSet("audit", "kernel", New, parse.EmptyHostParser); err != nil {
+		panic(err)
+	}
+}
+
+// MetricSet listens for audit messages from the Linux kernel using a netlink
+// socket. It buffers the messages to ensure ordering and then streams the
+// output. MetricSet implements the mb.PushMetricSet interface, and therefore
+// does not rely on polling.
+type MetricSet struct {
+	mb.BaseMetricSet
+	config Config
+	client *libaudit.AuditClient
+}
+
+// New constructs a new MetricSet.
+func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
+	logBetaOnce.Do(func() {
+		logp.Beta("The %v metricset is a beta feature", metricsetName)
+	})
```

Comment:
Is this really needed? I assume modules are not reloaded that often, so having this once in the log for each reload does not seem to bad? This "problem" applies to all metricsets with the beta log, rigth?

Can you change it to experimental?